MapboxCoreNavigation/RouteController.swift

```diff
@@ -355,10 +355,25 @@ open class RouteController: NSObject {
         locationManager.stopUpdatingHeading()
     }
     
+    /**
+     The most recently received user location. Snapped to the route line, if applicable, otherwise raw.
+     - seeAlso: snappedLocation, rawLocation
+     */
+    @objc public var location: CLLocation? {
+        return snappedLocation ?? rawLocation
+    }
+    
+    /**
+     The most recently recieved user location, snapped to the route line, if applicable.
+     - important: If the rawLocation is outside of the route snapping tolerances, this value is nil.
+     */
+    var snappedLocation: CLLocation? {
```

Comment:
Is this variable necessary? Can what's returned here be inlined into `self.location`?

Reply:
Replied [here.](https://github.com/mapbox/mapbox-navigation-ios/pull/1188#issuecomment-370519058)